onecgiar-pr-client/src/app/pages/ipsr/pages/innovation-package-detail/pages/ipsr-innovation-use-pathway/pages/step-n4/components/step-n4-bilateral-investment-table/modal/step-n4-edit-bilateral/step-n4-edit-bilateral.component.ts

```diff
@@ -1,26 +1,26 @@
 import { Component, DoCheck, Input, OnInit } from '@angular/core';
-import { BilateralexpectedinvestmentStep4, IpsrStep4Body } from '../../../../model/Ipsr-step-4-body.model';
-import { InstitutionsService } from 'src/app/shared/services/global/institutions.service';
-import { CentersService } from 'src/app/shared/services/global/centers.service';
-import { ApiService } from 'src/app/shared/services/api/api.service';
+import { BilateralexpectedinvestmentStep4 } from '../../../../model/Ipsr-step-4-body.model';
+import { InstitutionsService } from '../../../../../../../../../../../../shared/services/global/institutions.service';
+import { CentersService } from '../../../../../../../../../../../../shared/services/global/centers.service';
+import { ApiService } from '../../../../../../../../../../../../shared/services/api/api.service';
 
 @Component({
   selector: 'app-step-n4-edit-bilateral',
   templateUrl: './step-n4-edit-bilateral.component.html',
   styleUrls: ['./step-n4-edit-bilateral.component.scss']
 })
 export class StepN4EditBilateralComponent implements OnInit, DoCheck {
-  @Input() body: any;
+  @Input() body: any = {};
```

Comment:
@yecksin with @davinzifc we have decided not to allow the use of `any` as a type in the back-end. Please consider doing the same in the front-end, by adding the linter rule [no-explicit-any](https://typescript-eslint.io/rules/no-explicit-any)